src/main/java/org/folio/rs/client/EnrichHeadersClient.java

```diff
@@ -0,0 +1,49 @@
+package org.folio.rs.client;
+
+import static org.folio.spring.integration.XOkapiHeaders.TENANT;
+import static org.folio.spring.integration.XOkapiHeaders.TOKEN;
+
+import feign.Client;
+import feign.Request;
+import feign.Request.Options;
+import feign.Response;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.SneakyThrows;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.folio.spring.FolioExecutionContext;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@Log4j2
+public class EnrichHeadersClient extends Client.Default {
```

Comment:
This will be removed in the next PRs, as there is an alternative in spring-folio